fast64_internal/f3d/f3d_gbi.py

```diff
@@ -1789,20 +1794,14 @@ def vertexScrollTemplate(
         if scrollDataFields[0].animType == "None" and scrollDataFields[1].animType == "None":
             return ""
 
-        data = (
-            "void scroll_"
-            + name
-            + "() {\n"
-            + "\tint i = 0;\n"
-            + "\tint count = "
-            + str(count)
-            + ";\n"
-            + "\tint width = "
-            + str(fScrollData.dimensions[0])
-            + " * 0x20;\n"
-            + "\tint height = "
-            + str(fScrollData.dimensions[1])
-            + " * 0x20;"
+        data = "\n".join(
+            (
+                "void scroll_" + name + "() {",
+                "\tint i = 0;",
+                f"\tint count = {count};",
+                f"\tint width = {fScrollData.dimensions[0]} * 0x20;",
+                f"\tint height = {fScrollData.dimensions[1]} * 0x20;",
```

Comment:
ahhhh this is mostly a note to self, but this generated code can cause compiler warnings for unused variables, plus these aren't ever changed so they could be `const`

Reply:
this is a Python cleanup PR get your bad C out of here :japanese_goblin:

Reply:
lmaooo